ccdImports/configFiles/CCD_Probate_Backoffice/CaseEventToFields.json

```diff
@@ -628,7 +654,7 @@
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boAmendCaseDetailsForCaseCreated", "CaseFieldID": "ihtNetValue", "PageFieldDisplayOrder": 15, "DisplayContext": "MANDATORY", "PageID": "boAmendCaseDetailsPage2", "PageDisplayOrder": 2, "ShowSummaryChangeOption": "y"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boAmendCaseDetailsForCaseCreated", "CaseFieldID": "foreignAsset", "PageFieldDisplayOrder": 16, "DisplayContext": "OPTIONAL", "PageID": "boAmendCaseDetailsPage2", "PageDisplayOrder": 2, "ShowSummaryChangeOption": "y"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boAmendCaseDetailsForCaseCreated", "CaseFieldID": "foreignAssetEstateValue", "PageFieldDisplayOrder": 17, "DisplayContext": "OPTIONAL", "PageID": "boAmendCaseDetailsPage2", "PageDisplayOrder": 2, "FieldShowCondition": "foreignAsset=\"Yes\"", "ShowSummaryChangeOption": "y"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boAmendCaseDetailsForCaseCreated", "CaseFieldID": "solicitorIsMainApplicant", "PageFieldDisplayOrder": 1, "DisplayContext": "READONLY", "PageID": "boAmendCaseDetailsPage3", "PageDisplayOrder": 3, "FieldShowCondition": "applicationType=\"Solicitor\"", "PageShowCondition": "selectionList=\"applicant\"", "ShowSummaryChangeOption": "y"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boAmendCaseDetailsForCaseCreated", "CaseFieldID": "solicitorIsMainApplicant", "PageFieldDisplayOrder": 1, "DisplayContext": "READONLY", "PageID": "boAmendCaseDetailsPage3", "PageDisplayOrder": 3, "FieldShowCondition": "primaryApplicantAliasReason=\"DUMMY_VALUE_TO_HIDE_FIELD\"", "PageShowCondition": "selectionList=\"applicant\"", "ShowSummaryChangeOption": "y"},
```

Comment:
Could you use the FieldShowCondition = NEVER SHOW here as well?

Reply:
will swap out for state = nevershow - think thats what you mean - good shout!